mapstory/templates/layers/layer_data.html

```diff
@@ -114,17 +114,13 @@
                 <h3>Map Services</h3>
                 <div class="embed">WMS 
                     <i class="fa fa-question-circle ng-scope" tooltip-append-to-body="true" tooltip-placement="top" tooltip="OpenGIS® Web Map Service Interface Standard"></i>
-                  
-                    <input id="shareWMS" value ="http://{{ request.get_host }}/geoserver/geonode/{{ resource }}/wms?REQUEST=GetCapabilities&SERVICE=WMS&TILED=true&VERSION=1.1.1">
-                  
+                    <input id="shareWMS" value ="https://{{ request.get_host }}/geoserver/geonode/{{ resource }}/wms?REQUEST=GetCapabilities&SERVICE=WMS&TILED=true&VERSION=1.1.1"> 
```

Comment:
Should use request.protocol rather than explicitly using https so that it will be consistent in non-ssl environments.